### DISCUSSION TITLE
custom chatterino badge for donating

### DISCUSSION THREAD

**Discussion by killertofus** [Chatterino/chatterino2#3185] [Ideas] [answered]
A chatterino badge that you can customize after you donated

**Comment by Mm2PL** [comment]
I think this would create a lot of cancer in chat personally even more so that custom mod and VIP badges.

**Comment by apa420** [comment]
There are multiple issues/threads with people wanting custom chatterino badges which have been closed and tldr it's not happening.

**Comment by zneix** [comment, marked as answer]
Since this is not happening I'm locking the discussion. In the future refrain from opening Discussions / Issues for something that's been discussed in the past (multiple times) and denied.
